src/Magnum/Audio/Test/RendererTest.cpp

```diff
@@ -46,6 +66,63 @@ void RendererTest::debugError() {
     CORRADE_COMPARE(out.str(), "Audio::Renderer::Error::InvalidOperation\n");
 }
 
+void RendererTest::debugDistanceModel() {
+    std::ostringstream out;
+    Debug(&out) << Renderer::DistanceModel::Inverse;
+    CORRADE_COMPARE(out.str(), "Audio::Renderer::DistanceModel::Inverse\n");
+}
+
+void RendererTest::listenerOrientation() {
+    constexpr Vector3 up{1, 2, 3}, fwd{3, 2, 1};
+    Renderer::setListenerOrientation(fwd, up);
+    std::array<Vector3, 2> orientation = Renderer::listenerOrientation();
+
+    CORRADE_COMPARE(orientation[0], fwd);
+    CORRADE_COMPARE(orientation[1], up);
+}
+
+void RendererTest::listenerPosition() {
+    constexpr Vector3 pos{1, 3, 2};
+    Renderer::setListenerPosition(pos);
+
+    CORRADE_COMPARE(Renderer::listenerPosition(), pos);
+}
+
+void RendererTest::listenerVelocity() {
+    constexpr Vector3 vel{1, 3, 2};
+    Renderer::setListenerVelocity(vel);
+
+    CORRADE_COMPARE(Renderer::listenerVelocity(), vel);
+}
+
+void RendererTest::listenerGain() {
+    constexpr Float gain = 0.512;
```

Comment:
Please use `f` suffix on floats (MSVC otherwise complains *a lot* and sometimes even refuses to compile that code).